test/input.txt

```diff
@@ -96,13 +96,14 @@
   find bet
   # does not match if none have keyword
   find 23912039120
-  # matching should be case-sensitive
-  find betsy
 
   # find unique keyword
+  find betsy
   find Betsy
   # find multiple with same keyword
   find Dickson
+  #find contacts with non-private phone numbers containing keywords
+  find 222 444
```

Comment:
How about other tests? For example, according to the guide, finding both by name and number in the same command should work.